ui-tests/cypress/support/pageObjects/content/templates/TemplateForm.js

```diff
@@ -54,32 +67,13 @@ export default class TemplateForm extends AdminContent {
                .find(this.stylesheetInput);
   }
 
-  typeId(value) {
-    this.getIDInput().type(value);
-  }
-
-  typeName(value) {
-    this.getNameInput().type(value);
-  }
-
   selectContentType(value) {
-    this.getContentTypeDropdown().select(value);
+    return this.getContentTypeDropdown().then(input => this.select(input, value));
   }
 
   typeHTMLModel(value) {
     this.getContentShapeInput().click().type(value);
```

Comment:
this can be refactored as well to select the element and then click it and type on it using `Content` methods from the test code... why do we need to click on it before typing anyway?